public/resources/ts/calculate-player-stats.ts

```diff
@@ -40,6 +40,20 @@ export function getPlayerStats() {
     }
   }
 
+  // Active equipment stats
+  for (const piece of items.equipment) {
+    const bonusStats: ItemStats = helper.getStatsFromItem(piece as Item);
+
+    for (const [name, value] of Object.entries(bonusStats)) {
+      if (!allowedStats.includes(name)) {
+        continue;
+      }
+
+      stats[name].equipment ??= 0;
+      stats[name].equipment += value;
+    }
+  }
+
```

Comment:
do we ever display this?

Reply:
![image](https://user-images.githubusercontent.com/75372052/193359564-e358f5d3-dd65-49d6-bfcb-1589d61a8ea2.png)

Reply:
Cool